static/js/popovers.js

```diff
@@ -34,6 +34,32 @@ var list_of_popovers = [];
     }
 }($.fn.popover));
 
+function init_email_clipboard() {
+    var email_el = $('.my_email');
+    var copy_icon = $('.my_email i');
+
+    // if the user clicked from one popover to other
+    // there would be two copy_icon and its guarantted to the second
+    // element as its inserted after the first popover
+    var el_index = copy_icon.length !== 1 ? 1 : 0;
```

Comment:
Can you explain what's going on here?  This seems really hacky.

Reply:
If you click from on popover to another this feature breaks since the other popover is not removed from DOM. So think of it as There are two popover in DOM instead of one and the once that is first is one that's hidden. Does that make sense?

Reply:
I think we should use a more precise selector to catch the current popover, rather than indexing this way...